src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -179,30 +223,40 @@ To run a browser-level and protocol-level test concurrently, you can use [scenar
 <CodeGroup labels={["script.js"]} lineNumbers={[true]}>
 
 ```javascript
-import { chromium } from 'k6/experimental/browser';
+import { browser } from 'k6/experimental/browser';
 import { check } from 'k6';
 import http from 'k6/http';
 
 export const options = {
   scenarios: {
-    browser: {
+    testk6: {
       executor: 'constant-vus',
-      exec: 'browser',
+      exec: 'testk6',
```

Comment:
change this back to `browser`

Reply:
same: https://github.com/grafana/k6-docs/pull/1200#discussion_r1221394927

Reply:
Ah :( fair enough!